latex/delegation.tex

```diff
@@ -371,7 +526,8 @@ \subsection{Witnesses}
       & \var{cepoch} \vdash \var{pstate}
       \trans{pool}{c} \var{pstate'}
       \\ ~ \\
-      \verify{vk_s}{\serialised{(\dbody{c},~\txbody \var{tx})}}{\sigma}
+      \mathcal{V}_{\var{vk_s}}{\serialised{tx}}_{\sigma}
+      %\verify{vk_s}{\serialised{(\dbody{c},~\var{tx})}}{\sigma}
```

Comment:
the signature needs to include both the transaction body and the certificate bodies.

Reply:
Ok got it - I will put it back.